internal/interpreter/ffi.rs

```diff
@@ -876,6 +876,29 @@ pub struct PropertyDescriptor {
     property_type: ValueType,
 }
 
+/// CallbackDescriptor is a structure that's used to describe a callback declared in .slint
+/// code. It is returned from slint::interpreter::ComponentDefinition::callback_descriptors() and slint::interpreter::ComponentDefinition::global_callback_descriptors().
+#[derive(Clone)]
+#[repr(C)]
+pub struct CallbackDescriptor {
```

Comment:
This is duplicated here because the docs contain links to the ComponentDefinition functions, which can't be shared for Rust and C++ AFAICS :(